packages/dialtone-vue3/directives/scrollbar/scrollbar.js

```diff
@@ -0,0 +1,17 @@
+import { OverlayScrollbars } from 'overlayscrollbars';
+import 'overlayscrollbars/overlayscrollbars.css';
+
+export const DtScrollbarDirective = {
+  name: 'dt-scrollbar-directive',
+  install (app) {
+    app.directive('dt-scrollbar', {
+      mounted (el) {
+        OverlayScrollbars(el, { scrollbars: { autoHide: 'scroll' } });
```

Comment:
I'm not sure where to add this comment, so I just chose this line...

Can we default to always show the scrollbar on hover of the scrollable region? And then provide doc that shows how to change that prop to whatever its options area?

Reply:
Yes, that can be done 👍